crates/bevy_ecs/src/archetype.rs

```diff
@@ -254,31 +355,57 @@ impl Archetype {
             .map(|(id, _)| *id)
     }
 
+    /// Gets an iterator of all of the components in the archetype.
+    ///
+    /// All of the IDs are unique.
     #[inline]
     pub fn components(&self) -> impl Iterator<Item = ComponentId> + '_ {
         self.components.indices()
     }
 
+    /// Fetches a immutable reference to the archetype's [`Edges`], a cache of
+    /// archetypal relationships.
     #[inline]
     pub fn edges(&self) -> &Edges {
         &self.edges
     }
 
+    /// Fetches a mutable reference to the archetype's [`Edges`], a cache of
+    /// archetypal relationships.
     #[inline]
     pub(crate) fn edges_mut(&mut self) -> &mut Edges {
         &mut self.edges
     }
 
+    /// Fetches the row in the [`Table`] where the components for the entity at `index`
+    /// is stored.
+    ///
+    /// An entity's archetype index can be fetched from [`EntityLocation::index`], which
+    /// can be retrieved from [`Entities::get`].
+    ///
+    /// # Panics
+    /// This function will panic if `index >= self.len()`.
+    ///
+    /// [`Table`]: crate::storage::Table
+    /// [`EntityLocation`]: crate::entity::EntityLocation::index
+    /// [`Entities::get`]: crate::entity::Entities::get
     #[inline]
     pub fn entity_table_row(&self, index: usize) -> usize {
         self.entities[index].table_row
     }
 
+    /// Updates where an the components for the entity at `index` can be found
```

Comment:
Updates where an the components -> Updates if the components

Reply:
```suggestion
    /// Updates if the components for the entity at `index` can be found
```